OPHD/States/MapViewState.cpp

```diff
@@ -268,6 +268,20 @@ int MapViewState::refinedResourcesInStorage()
 }
 
 
+void MapViewState::updatePlayerResources()
+{
+	StructureList storage = NAS2D::Utility<StructureManager>::get().structureList(Structure::StructureClass::Storage);
+	storage.insert(storage.begin(), mTileMap->getTile(ccLocation()).structure());
```

Comment:
It could, but in this case I'm running on the assumption that if there's more than one command center we're only interested in the main command center. I'm sort of forward thinking here. Also, this ensures that the command center is always the first in the list since it acts as a sort of a backup system should something in the colony fail allowing the player to recover versus having an immediate collapse.

Reply:
Hmm, but shouldn't all command center's provide the same backup storage?